static/js/admin/src/URLUtilities.js

```diff
@@ -58,6 +58,18 @@ define('URLUtilities', ['jquery'], function($) {
 			if (newParameterString)
 				return baseURL + "?" + newParameterString;
 			return baseURL;
+		},
+
+		getBaseUrl: function(url, access) {
+			if (access === undefined || !access) {
+				return url.substring(0, url.indexOf("/admin/") + 7);
```

Comment:
not a huge deal, but wonder whether using  length() function here instead of 7 would make for code that is better self-documenting

Reply:
Updated in new commit